docs/source/azure/policy/resources/vm.rst

```diff
@@ -0,0 +1,84 @@
+.. _azure_vm:
+
+Virtual Machines
+================
+
+Filters
+-------
+- Standard Value Filter (see :ref:`filters`)
+
+``instance-view``
+  Filter based on VM attributes
+
+  .. c7n-schema:: InstanceViewFilter
+       :module: c7n_azure.resources.vm
+
+Actions
+-------
+
+``start``
+  Start the VMs
```

Comment:
On each of these you can probably include the schema automatically - I know it isn't much of one but it can just be a standard format for listing actions and filters (if it isn't finding anything we may need to go define the schema on the actions?)

```
.. c7n-schema:: MetricFilter
    :module: c7n_azure.filters
```

Reply:
@stefangordon when I add these for each of the VM actions, they show up as blank. I went to look at the schema definition, and I'm using the `type_schema()` method from `c7n.filters.core`. Should I just be defining them with a string? It looks like those show up just fine in the documentation